### PR TITLE
Merge the python and pandas guides

### PR DESCRIPTION
**Pull request by JesterOrNot**
As discussed previously discussed.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -53,6 +53,11 @@
   to = "/docs/config-docker/"
   status = 301
 
+[[redirects]]
+  from = "/docs/languages/pandas"
+  to = "/docs/languages/python#pandas"
```
Comment (jankeromnes) [inline netlify.toml:58]: Redirect to canonical URL:

```suggestion
  to = "/docs/languages/python/#pandas"
```

File: src/docs/languages/python.md [hunk 2]
```diff
@@ -152,6 +152,27 @@ To see a basic repository with Python debugging enabled, please check out [gitpo
 
 <br>
 
+## Pandas
+
+Welcome data scientists! This part of the guide will show you how to configure Gitpod for Pandas development.
+
+## Start Tasks
```
Comment (jankeromnes) [inline src/docs/languages/python.md:159]: Fix sub-title level:

```suggestion
### Start Tasks
```

File: src/docs/languages/python.md [hunk 3]
```diff
@@ -152,6 +152,27 @@ To see a basic repository with Python debugging enabled, please check out [gitpo
...
+
+Setting up [start tasks](/docs/config-start-tasks/) for Pandas isn't very different from other Python frameworks, here is an example of how to configure you start tasks:
+
+```yaml
+tasks:
+    # Install requirements in this case pandas
+  - init: pip3 install -r requirements.txt
+    # Run your program on workspace start
+    command: python3 <PATH TO APP>
+```
```
Comment (jankeromnes) [inline src/docs/languages/python.md:169]: Isn't this almost exactly the same thing as the Python "Start Tasks" section? Why duplicate it here?

File: src/docs/languages/python.md [hunk 4]
```diff
@@ -152,6 +152,27 @@ To see a basic repository with Python debugging enabled, please check out [gitpo
...
+
+## Try it
```
Comment (jankeromnes) [inline src/docs/languages/python.md:171]: Fix sub-title level, make it less ambiguous:

```suggestion
## Try Pandas in Gitpod
```

File: src/docs/languages/python.md (hunk 3 above)
Comment (jankeromnes) [reply src/docs/languages/python.md:169]: I think you can remove this entire "Start Tasks" sub-section.

Comment (JesterOrNot) [reply src/docs/languages/python.md]: K, done!

**Comment by JesterOrNot** [comment]
Thanks, Merging!

**Pull request merged**